suites/portal/fileExplorerTest.js

```diff
@@ -0,0 +1,56 @@
+Feature('File Explorer');
+
+/*
+Data Setup:
+    1. Commons_1 - URL configured
+        i) User_11 - No access to some data
+        ii) User_12 - Access to all data
+    2. Commons_2 - URL not configured
+        i) User_21 - No access to some data
+        ii) User_22 - Access to all data
+ */
```

Comment:
Not a blocker, but is there a way to make this less complex?

Reply:
@haraprasadj , perhaps you could add a sample URL.
Take, per instance, the Guppy-related tests that have been merged recently:
https://github.com/uc-cdis/gen3-qa/pull/183/files
It has lines like:
> "+        Take qa-mickey as example:"
> "+        1. Goto target commons' query page (e.g., https://qa-mickey.planx-pla.net/query)."

Reply:
@atharvar28 will describe the data a bit more in the scenario (added as a separate section to help in automation later)
@themarcelor will do that